src/app/components/DefinitionList/DefinitionList.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { themeSpacing } from '@datapunt/asc-ui'
+import styled from 'styled-components'
+
+const DefinitionListWrapper = styled.ul`
+  width: 100%;
+  margin-bottom: ${themeSpacing(2)};
+`
+
+const DefinitionList: React.FC<{}> = ({ children }) => (
```

Comment:
If the component takes no props you can also drop the generic type to `React.FC`.

```suggestion
const DefinitionList: React.FC = ({ children }) => (
```